pkg/daemon/daemon.go

```diff
@@ -1025,7 +1008,6 @@ func checkFileContentsAndMode(filePath string, expectedContent []byte, mode os.F
 
 // Close closes all the connections the node agent has open for it's lifetime
 func (dn *Daemon) Close() {
-	dn.loginClient.Close()
```

Comment:
question: it's ok that this func is empty?

Reply:
yup, just doing nothing/returning nothing - as for keeping it, I don't have a strong opinion